src/css/style.css

```
@@ -1,14 +1,17 @@
 @font-face {
   font-family: "quan";
   src: url(../assets/font/Quan-Book.otf);
+  font-display: swap;
 }
 @font-face {
   font-family: "quan-light";
```

Stylelint (17.3.0)

[error] Unexpected quotes around "quan-light" (font-family-name-quotes)

(font-family-name-quotes)